packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -171,6 +175,13 @@ export interface GitHubSourceProps extends BuildSourceProps {
    */
   oauthToken: cdk.Secret;
 
+  /**
+   * Whether to create a webhook that will trigger a build every time a commit is pushed to the GitHub repository.
+   *
+   * @default true
```

Comment:
true? Not sure this makes sense as the default...

Reply:
Sure, will change to `false`.